### DISCUSSION TITLE
Tabs for opened connections

### DISCUSSION THREAD

**Discussion by egolfeld** [isontheline/pro.webssh.net#478] [Ideas]
Probably it will much easier to navigate between each ssh connection

**Comment by isontheline** [comment]
Tabs instead of [Multiple Terminals feature?](https://webssh.net/documentation/help/howtos/launching-multiple-terminals/)

**Reply by egolfeld** [reply]
Sort of it, multiple terminals feature is good solution, but users have to click twice, sorry, maybe I shouldn't be so lazy, but tapping on tab can be much faster, and can save lot if time if user work with several connections at same time

**Reply by isontheline** [reply]
Tabs are eating a lot of space 😢

Instead couldn't we use keyboard shortcuts to switch between terminals? (In case of external keyboard)

Anyway, tabs support is already requested here : https://github.com/isontheline/pro.webssh.net/issues/101

**Reply by egolfeld** [reply]
Hm..screen, I am using ipad, so it's not a problem on it, but on iphones..agree 

Maybe keyword shortcuts interesting alternative, with some toast to easy identification of connection

**Reply by egolfeld** [reply]
Up to you, just a suggestion :)

**Reply by isontheline** [reply]
All suggestions are welcome 😇 

WebSSH is built for everyone who need to work efficiently and I agree that multiple windows management is not as great right now.

There are several users wanting tabs, so I will search for the upcoming releases a way to add them without penalizing iPhone screens too.

Thank you a lot for your feedback 🙏

**Comment by JellyFisch** [comment]
I Love your SSH Client -> and this is the last thing i need, this Feature is Beautiful and thx that you work on it.

**Comment by isontheline** [comment]
Thanks for your feedback @JellyFisch 🙏

Which device are you using? iPhone / iPad / macOS
